src/plugins/sync-plugin.js

```diff
@@ -965,6 +1000,17 @@ const marksToAttributes = (marks) => {
   return pattrs
 }
 
+const nodeMarksToAttributes = (marks) => {
+  const pattrs = {}
+  const prefix = 'yelement_mark_'
```

Comment:
Let's store this prefix globally somewhere as it is reused throughout the file. Also I suggest we use a shorter name that starts with an `_`. E.g. `_mark_`.